src/main/java/org/zalando/nakadi/service/subscription/SubscriptionService.java

```diff
@@ -119,11 +122,10 @@ public SubscriptionService(final SubscriptionDbRepository subscriptionRepository
     public Subscription createSubscription(final SubscriptionBase subscriptionBase)
             throws TooManyPartitionsException, RepositoryProblemException, DuplicatedSubscriptionException,
             NoSuchEventTypeException, InconsistentStateException, WrongInitialCursorsException,
-            DbWriteOperationsBlockedException, UnableProcessException, ServiceTemporarilyUnavailableException {
-        if (featureToggleService.isFeatureEnabled(FeatureToggleService.Feature.DISABLE_DB_WRITE_OPERATIONS)) {
-            throw new DbWriteOperationsBlockedException("Cannot create subscription: write operations on DB " +
-                    "are blocked by feature flag.");
-        }
+            DbWriteOperationsBlockedException, UnableProcessException,
+            AuthorizationSectionException, ServiceTemporarilyUnavailableException {
+
+        checkFeatureToggles(subscriptionBase);
```

Comment:
I like that you move the checks for feature toggles to their own private method, it removes clutter.
However, I would suggest to be more precise in the naming: you're actually checking feature toggles that are relevant to the creation and update of subscriptions specifically, and this should be reflected in the method name.